_posts/2021-08-22-release-2.0.0.md

```diff
@@ -0,0 +1,162 @@
+- TPRParser now loads TPR files with `tpr_resid_from_one=True` by default, which starts TPR resid indexing from 1 (instead of 0 as in previous MDAnalysis versions) (Issue [#2364][], PR [#3152][])
+- `analysis.hole` has now been removed in favour of `analysis.hole2.hole` (Issue [#2739][])
+- `Writer.write(Timestep)` and `Writer.write_next_timestep` have been removed. Please use `write()` instead (Issue [#2739][])
+- Removes deprecated density_from_Universe, density_from_PDB, Bfactor2RMSF, and notwithin_coordinates_factory from MDAnalysis.analysis.density (Issue [#2739][])
+- Removes deprecated `waterdynamics.HydrogenBondLifetimes` (PR [#2842][])
+- `hbonds.WaterBridgeAnalysis` has been moved to `hydrogenbonds.WaterBridgeAnalysis` (Issue [#2739][] PR [#2913][])
+
+## Deprecations
+- The `bfactors` attribute is now aliased to `tempfactors` and will be removed in 3.0.0 (Issue [#1901][])
+- `WaterBridgeAnalysis.generate_table()` now returns table information, with the `table` attribute being deprecated
+- Various analysis result attributes which are now stored in `Results` will be deprecated in 3.0.0 (Issue [#3261][])
+- In 3.0.0 the ParmEd classes will only be accessible from the `MDAnalysis.converters` module
+- In 2.1.0 the TRZReader will default to a dt of 1.0 ps when failing to read it from the input TRZ trajectory
+
+## Author statistics
+
+Altogether, this represents the work of 35 authors. 17 of which were new contributors:
+
+ - @daveminh
+ - @mieczyslaw
+ - @aditya-kamath
+ - @tirkarthi
+ - @LeonardoBarneschi
+ - @hejamu
+ - @biogen98
+ - @orioncohen
+ - @z3y50n
+ - @hp115
+ - @ojeda-e
+ - @thadanipaarth
+ - @HenryKobin
+ - @1ut
+ - @sulays
+ - @ahy3nz
+ - @fiszczyp
+
+MDAnalysis thanks [NumFOCUS][]'s continued support as the organisation's fiscal sponsor.
+
+- The MDAnalysis Team
+
+[roadmap]: {{ site.baseurl }}{% post_url 2019-11-06-roadmap %}
+[conda-forge channel]: https://anaconda.org/conda-forge/mdanalysis
+[PyPi]: https://pypi.org/project/MDAnalysis/
+[installation instructions in the User Guide]: https://userguide.mdanalysis.org/stable/installation.html
+[CHANGELOG]: https://github.com/MDAnalysis/mdanalysis/blob/release-2.0.0/package/CHANGELOG
+[Google Summer of Code]: https://summerofcode.withgoogle.com/
+[RDKit converter]: https://docs.mdanalysis.org/2.0.0/documentation_pages/converters/RDKit.html
+[ChEMBL27]: https://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLdb/releases/chembl_27/
+[OpenMM converter]: https://docs.mdanalysis.org/2.0.0/documentation_pages/converters/OpenMM.html
+[parallel analysis]: https://userguide.mdanalysis.org/stable/examples/analysis/custom_parallel_analysis.html
+[Mean Square Displacement]: https://docs.mdanalysis.org/2.0.0/documentation_pages/analysis/msd.html
+[Results class]: https://docs.mdanalysis.org/2.0.0/documentation_pages/analysis/base.html?highlight=results#MDAnalysis.analysis.base.AnalysisBase.results
```

Comment:
remove highlight?

Reply:
updated to point at the actual Results class docs